x/leveragelp/types/expected_keepers.go

```diff
@@ -79,3 +79,7 @@ type MasterchefKeeper interface {
 	ClaimRewards(ctx sdk.Context, sender sdk.AccAddress, poolIds []uint64, recipient sdk.AccAddress) error
 	UserPoolPendingReward(ctx sdk.Context, user sdk.AccAddress, poolId uint64) sdk.Coins
 }
+
+type AccountedPoolKeeper interface {
+	GetAccountedBalance(sdk.Context, uint64, string) math.Int
+}
```

Comment:
I don't think we are using it, if not let's remove this

Reply:
got it, so we use to pass values, why not use `k.ExitPoolEst` directly ? we can avoid this extra dep here

Reply:
`k.ammKeeper.ExitPoolEst`